src/event/containers/PastEvents.tsx

```diff
@@ -46,12 +42,11 @@ interface PastEventsContainerProps {
   ];
   fetchAttendance: Function;
   fetchPastEvents: Function;
-  updateTimeframe: Function;
-  timeframe: string;
 }
 
 const PastEventsContainer: React.FC<PastEventsContainerProps> = (props) => {
-  const { auth, events, attendance, fetchAttendance, fetchPastEvents, timeframe, updateTimeframe } = props;
+  const { auth, events, attendance, fetchAttendance, fetchPastEvents } = props;
+  const [timeframe, setTimeframe] = useState(getCurrentYear()?.name ?? 'All Time');
```

Comment:
when would `getCurrentYear` fail?

Reply:
When you pass in an academic year not currently in the dataset of the package; you can check out the available academic years [here](https://github.com/acmucsd/ucsd-quarters-years/blob/main/src/data.ts).